src/types/currency.ts

```diff
@@ -36,10 +36,14 @@ export type CollateralIdLiteral =
 const CollateralCurrency = [Polkadot, Kusama, Interlay, Kintsugi] as const;
 type CollateralCurrency = typeof CollateralCurrency[number];
 
-export type ForeignAsset = Currency & {foreignAsset: { id: number; coingeckoId: string }};
-export type LendToken = Currency & {lendToken: { id: number }}; 
+export type ForeignAsset = Currency & { foreignAsset: { id: number; coingeckoId: string } };
+export type LendToken = Currency & { lendToken: { id: number } };
+export type StandardLPToken = Currency & { lpToken: { token0: CurrencyExt; token1: CurrencyExt } };
```

Comment:
I think if we want to avoid the circular reference here.
`StandardLpToken` can contain up to 2 `CurrencyExt` which can be an `LpToken` which can be a `StandardLPToken`

I was under the impression `tokenX` can only be a native token, foreign asset, or stable pool id.
See type definition in [interbtc-types definitions](https://github.com/interlay/interbtc-types/blob/3d4736fb3a1ca6b51ace54f8995134d3d98d9690/definitions.json#L12-L27)

Reply:
This is good suggestion, actually the runtime uses lowercase `p` as well, so I changed it :+1:

Reply:
Thanks for noticing this :pray: